tasks/bundle-platform

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
Can you name this file `bundle_platform.rb` to keep in line with the other tasks?